pkg/uprobetracer/ldcache_parser.go

```diff
@@ -0,0 +1,199 @@
+package uprobetracer
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"os"
+	"strings"
+	"unsafe"
+)
+
+const (
+	cache1Header    string = "ld.so-1.7.0"
+	cache1HeaderLen int    = 11
```

Comment:
This can be removed and replaced by `len(cache1Header)`.

Ditto for `cache2HeaderLen`.